### PR TITLE
macvtap: Use podman on postsubmit jobs

### PR DESCRIPTION
**Pull request by oshoval**
Signed-off-by: Or Shoval <oshoval@redhat.com>

### CODE REVIEW

**Comment by oshoval** [comment]
@brianmcarey 
Is there a way to test postsubmit before merging it ? (not sure there is)

**Comment by brianmcarey** [comment]
> @brianmcarey Is there a way to test postsubmit before merging it ? (not sure there is, unless running manual on CI or so)

There isn't really as far as I know. . The hope would be that the majority actions in a postsubmit job would be covered by presubmits in the repo so if the update works in the presubmits - it should work in the postsubmit. 

@dhiller is there a method of testing postsubmits that I am missing?

**Review by brianmcarey — COMMENTED** [review]
/lgtm

**Comment by oshoval** [comment]
Hi @maiqueb 
can you take a look please as well ?

**Review by maiqueb — CHANGES_REQUESTED** [review]

File: github/ci/prow-deploy/files/jobs/kubevirt/macvtap-cni/macvtap-cni-postsubmits.yaml [hunk 1]
```diff
@@ -31,14 +34,17 @@ postsubmits:
...
         preset-github-credentials: "true"
         preset-kubevirtci-quay-credential: "true"
       cluster: ibm-prow-jobs
       spec:
         containers:
-          - image: quay.io/kubevirtci/bootstrap:v20220728-1410a63
+          - image: quay.io/kubevirtci/bootstrap:v20220823-3fed276
+            env:
+            - name: OCI_BIN
+              value: podman
```
[the remark this answers is not in the document]
Comment (oshoval) [reply github/ci/prow-deploy/files/jobs/kubevirt/macvtap-cni/macvtap-cni-postsubmits.yaml:47]: We run at `automation/release.sh`, this command `make release`
release depends on `release: docker-build docker-push`, which depends on `OCI_BIN`
```
docker-build:
	$(OCI_BIN) build -t ${IMAGE_REGISTRY}/${IMAGE_NAME}:${IMAGE_TAG} -f ./cmd/Dockerfile .
```

[the remark this answers is not in the document]
Comment (maiqueb) [reply github/ci/prow-deploy/files/jobs/kubevirt/macvtap-cni/macvtap-cni-postsubmits.yaml:47]: Oh. You're right ... while the release make target does [**not** require](https://github.com/kubevirt/macvtap-cni/blob/main/hack/release.sh) the `OCI_BIN` variable, its dependencies [do](https://github.com/kubevirt/macvtap-cni/blob/c40e78241abd048d322952b93ed42d80ea258979/Makefile#L125).

**Review by maiqueb — APPROVED** [review]
I stand corrected. 

@phoracek I wonder if it makes sense to have the release make target depend on img building / img generation: it releases the src code ... as long as it builds (and the tests pass ...) we should be able to release the src code, right ?

**Comment by phoracek** [comment]
@maiqueb you mean to make sure we test before we push the images to public? That makes sense

**Comment by maiqueb** [comment]
> @maiqueb you mean to make sure we test before we push the images to public? That makes sense

I unfortunately don't have room to take any task right now. I agree it makes sense, I just can't make up room for this.

**Comment by maiqueb** [comment]
> > @maiqueb you mean to make sure we test before we push the images to public? That makes sense
> 
> I unfortunately don't have room to take any task right now. I agree it makes sense, I just can't make up room for this.

Furthermore, what would the "test" look like ? I am practically illiterate on this repo.

**Pull request merged**